bot/exts/utils/utils.py

```diff
@@ -40,6 +40,7 @@
 If the implementation is easy to explain, it may be a good idea.
 Namespaces are one honking great idea -- let's do more of those!
 """
+LEADS_AND_COMMUNITY = (Roles.project_leads, Roles.domain_leads, Roles.partners, Roles.python_community)
```

Comment:
Leads could be made a separate constant like `MODERATION_ROLES` in `constants.py`, same for community.

Reply:
If we do eventually use these roles more often, then we probably should.

At this current moment, `LEADS_AND_COMMUNITY` contains roles that have never been used anywhere else.